### PR TITLE
[Issue #218] StdOut was not being redirected properly and was causing…

### PR DESCRIPTION
**Pull request by pnikonowicz**
… the child process to hang.

When this is merged, we should create a new task to include logging of stdout properly using the Log4Net feature that was recently completed.

### CODE REVIEW

**Comment by pnikonowicz** [comment]
build is failing because .Net 4+ features were used and are incompatible with .Net 2.0. 
Will fix soon.

**Comment by pnikonowicz** [comment]
build should be fixed now. if so, please merge.

**Comment by oleg-nenashev** [comment]
IIRC the plan in #218 was to actually make it working. OTOH it can be applicable as a hotfix. I will try to test and merge something this week. Was traveling, so missed the PR.

Please also revert the permission change in the file. There is no reason for these files to be executable

**Comment by pnikonowicz** [comment]
Yes, the idea was to do a hot fix on the main problem, and then create a new task for the remaining work. I tried to explain that in the PR description, sorry if that wasn't clear. 

I've been busy this week but will address the file perms soon.

File: src/Core/WinSWCore/Util/ProcessHelper.cs [hunk 1]
```diff
@@ -138,9 +138,9 @@ public static void StopProcessAndChildren(int pid, TimeSpan stopTimeout, bool st
             ps.WorkingDirectory = workingDirectory ?? ps.WorkingDirectory;
             ps.CreateNoWindow = false;
             ps.UseShellExecute = false;
-            ps.RedirectStandardInput = true; // this creates a pipe for stdin to the new process, instead of having it inherit our stdin.
-            ps.RedirectStandardOutput = true;
-            ps.RedirectStandardError = true;
+            ps.RedirectStandardInput = false; // this creates a pipe for stdin to the new process, instead of having it inherit our stdin.
```
Comment (oleg-nenashev) [inline src/Core/WinSWCore/Util/ProcessHelper.cs]: I'd guess this comment is also outdated now

**Comment by pnikonowicz** [comment]
file perms and comment issues have been fixed.

**Comment by pnikonowicz** [comment]
I would like to work on some more items but would like to see this PR closed or merged before that happens.

**Comment by oleg-nenashev** [comment]
Will get it integrated tonight. Sorry, was traveling

**Pull request merged**